src/config.ts

```diff
@@ -95,3 +100,15 @@ export const CONFIG: Config =
     .reduce((res, e) => {
       return { ...res, [e[0]]: e[1] };
     }, DEFAULT_CONFIG) ?? DEFAULT_CONFIG;
+
+export const PROMOTION_CONFIG: Config =
+  {
+    ...promotionSheet
+      ?.getRange('A2:B')
+      .getDisplayValues()
+      .filter(e => e[0])
+      .reduce((res, e) => {
+        return { ...res, [e[0]]: e[1] };
+      }, DEFAULT_CONFIG),
```

Comment:
Instead of using the DEFAULT_CONFIG here, I would use `CONFIG`, in this way the promo config sheet can have only the fields that need to be overwritten (other config params will be copied from the main config).

Reply:
same here, fixed in the new commit